apps/web/templatetags/form_tags.py

```diff
@@ -5,8 +5,8 @@
 
 
 @register.simple_tag
-def render_form_fields(form):
-    rendered_values = [render_field(form[field]) for field in form.fields]
+def render_form_fields(form, *fields):
+    rendered_values = [render_field(form[field]) for field in form.fields if not fields or field in fields]
```

Comment:
this is a funny line. Reminds me of the afrikaans sentence: "Wat was was voordat was was was?"